### ISSUE TITLE
Accessing accessibility zoom doesn't work

### ISSUE THREAD

**Issue by estatematt** [folivoraAI/BetterTouchTool#1896]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at boastr.net@gmail.com instead.

---

#### Description of bug/feature request/question:
command option equals does not zoom 

#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):
touch bar

#### Device information:

Type of Mac: 2016 17" MBP
macOS version:  10 12 6
BetterTouchTool version: 2.319 

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):
I can't figure out what application to select.  Not using any  predefined action doesn't work either

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.
